android/assets/jsons/Tutorials.json

```diff
@@ -168,5 +168,53 @@
 "ⓨ: By the way, here's how an empire border looks like - it's in the national colours of the nation owning the territory.",
 "ⓩ: And this is the red targeting circle that led to the attack pane back under ⑬.",
 "What you don't see: The phone/tablet's back button will pop the question whether you wish to leave Unciv and go back to Real Life. On desktop versions, you can use the ESC key."
-  ]
+  ],
+    "Faith": [
+        "After building a shrine, you're civilization will start generating faith.",
+        "When enough faith has been generated, you will be able to found a pantheon.",
+        "A pantheon will provide a small bonus for your civilization that will apply to all your cities.",
+        "Each civilization can only choose a single pantheon belief, and each pantheon can only be chosen once.",
+        "Generating more faith will allow you to found a religion."
+    ],
+    "Religion": [
+        "When you have gathered enough religion, a great prophet will be born in one of your cities.",
```

Comment:
enough faith? By the way, the Fonts.faith symbol will work, even if you don't have access to the name from json